pom.xml

```diff
@@ -167,6 +167,7 @@
     <exec-maven-plugin.version>3.1.1</exec-maven-plugin.version>
     <maven-checkstyle-plugin.version>3.3.0</maven-checkstyle-plugin.version>
     <mvel2.version>2.5.2.Final</mvel2.version>
+    <spotless.version>2.27.2</spotless.version>
```

Comment:
This is a pretty old version (from 2022). Is there a reason you're not using the latest? (2.43.0)

Reply:
Ah yes you are right. I just kept this in sync with hbase but yes let me bump into hbase codebase as well and move to the latest one over here as well.